services/cards-consultation/src/main/java/org/opfab/cards/consultation/configuration/webflux/IpAddressAuthorizationManager.java

```diff
@@ -62,8 +62,8 @@ public Mono<AuthorizationDecision> check(Mono<Authentication> authenticationMono
 
     private User getAuthenticatedUser(Authentication authentication) {
         User userData = null;
-        if (authentication.getPrincipal()  instanceof CurrentUserWithPerimeters){
-            CurrentUserWithPerimeters current = (CurrentUserWithPerimeters) authentication.getPrincipal();
+        if (authentication.getPrincipal()  instanceof CurrentUserWithPerimeters currentUserWithPerimeters){
+            CurrentUserWithPerimeters current = currentUserWithPerimeters;
             userData = current.getUserData();
```

Comment:
Is it necessary to define 'current' variable? Could we just use "userData = currentUserWithPerimeters .getUserData()" ?

Reply:
You're right. Done